alerts/apps_alerts.libsonnet

```
@@ -235,7 +235,7 @@
             'for': '15m',
           },
           {
-            alert: 'KubeJobCompletion',
+            alert: 'KubeJobIncomplete',
```

```suggestion
            alert: 'KubeJobNotCompleted',
```

`Incomplete` sounds like the Job is missing something.

---

I like `KubeJobNotCompleted` too